Pony/include/pony/Address.hpp

```diff
@@ -6,37 +6,41 @@
 /* Original story "Networking for Game Programmers" by Glenn Fiedler
  */
 
-#ifndef PONY_ADDRESS_HPP
-#define PONY_ADDRESS_HPP
-
-#include <pony/SharedMacros.hpp>
+#ifndef ___PONY__ADDRESS_HPP___
+#define ___PONY__ADDRESS_HPP___
 
 namespace pony
 {
-
-class PONY_EXPORT Address
+/**
+ * @brief Address
+ *
+ * IP Address/Port pair.
+ */
+class Address
 {
-
 public:
     Address();
-    Address(unsigned int address);
-    Address(unsigned int address, unsigned short port);
-    Address(unsigned char fst, unsigned char snd, unsigned char thd, unsigned char fth, unsigned short port);
+    Address(const unsigned int address);
+    Address(const unsigned int address, const unsigned short port);
+    Address(const unsigned char fst, const unsigned char snd, const unsigned char thd, const unsigned char fth, const unsigned short port);
 
     unsigned int GetAddress() const;
```

Comment:
Better use `uint32_t` or anything with a defined size. Unless there is a specific reason not to.

Reply:
Same goes for every type, for which standard defines only maximum size.